src/forklift/core.py

```diff
@@ -136,7 +136,7 @@ def remove_temp_table(table):
                                                                  crate.name,
                                                                  source_describe.shapeType.upper(),
                                                                  crate.source,
-                                                                 spatial_reference=source_describe.spatialReference)
+                                                                 spatial_reference=source_describe.spatialReference).getOutput(0)
```

Comment:
can we destructure this? 

[Result outputs can also be accessed by index, so result.getOutput(0) and result[0] are equivalent.](http://resources.arcgis.com/en/help/main/10.1/index.html#//018z00000046000000)